tests/integration/test_export_and_heudiconv.py

```diff
@@ -51,15 +72,15 @@ def test_dicom_export():
 
     # ***************************************************************************
     # Test that default overwrite flag is NOT wiping the xnat-export directory
+    # Here we test using typer's CLIRunner
```

Comment:
Why are we calling the function directly and using `CLIRunner`? Is there a situation in which we call the function directly?